apps/cowswap-frontend/src/modules/limitOrders/updaters/AlternativeLimitOrderUpdater.ts

```diff
@@ -0,0 +1,172 @@
+import { useSetAtom } from 'jotai'
```

Comment:
Do we really need to duplicate the updater? 
if we do in the future SWAP (maybe not super soon, but is possible), it feels we have way too many updaters, and I can see the updater explosion with alternative for all flows

Reply:
This one is what sets up the state, so it's not entirely a duplication.